File: Day288/buildAnArrayWithStackOperations.java
class Solution {
    public List<String> buildArray(int[] target, int n) {
        int s = 1;
        List<String> answer = new ArrayList<>();
        for (int v : target) {
            while (s < v) {
                answer.add( "Push");
                answer.add( "Pop");
                s++;
            }
            s++;
            answer.add( "Push");
        }
        return answer;
    }
}

File: Day288/buildAnArrayWithStackOperations.go
func buildArray(target []int, n int) []string {
	s := 1
	answer := make([]string, 0)
	for _,v := range target {
		for s < v {
			answer = append(answer, "Push")
			answer = append(answer, "Pop")
			s++
		}
		s++
		answer = append(answer, "Push")
	}
	return answer
 }